### ISSUE TITLE
Autocomplete not working for oh-my-zsh

### ISSUE THREAD

**Issue by dprothero** [oclif/plugin-autocomplete#8]
Have tried to use autocomplete on a Mac with oh-my-zsh installed:

```
➜  cli git:(master) printf "$(twilio autocomplete:script zsh)" >> ~/.zshrc; source ~/.zshrc
➜  cli git:(master) twilio <TAB>_twilio:6: command not found: Twilio.CURRENT
_twilio:6: command not found: Twilio.CURRENT src/           test/
_twilio:6: command not found: CompositionSid
_twilio:6: command not found: RecordingSid
_twilio:6: command not found: currently
_twilio:6: command not found: TrackNameOrSid

zsh: do you wish to see all 461 possibilities (462 lines)?
```

On Linux, I get something similar (testing with this [docker image](https://hub.docker.com/r/exadra37/node-zsh)):

```
➜  node printf "$(twilio autocomplete:script zsh)" >> ~/.zshrc; source ~/.zshrc
➜  node twilio <TAB>local:6: command not found: Twilio.CURRENT
local:6: command not found: Twilio.CURRENT
local:6: command not found: CompositionSid
local:6: command not found: RecordingSid
local:6: command not found: currently
local:6: command not found: TrackNameOrSid

local:6: unknown file attribute: \n
local:7: command not found: Twilio.CURRENT
local:7: command not found: Twilio.CURRENT
local:7: command not found: CompositionSid
local:7: command not found: RecordingSid
local:7: command not found: currently
local:7: command not found: TrackNameOrSid
local:7: unknown file attribute: \n
local:1: command not found: Twilio.CURRENT
local:1: command not found: Twilio.CURRENT
local:1: command not found: CompositionSid
local:1: command not found: RecordingSid
local:1: command not found: currently
local:1: command not found: TrackNameOrSid
local:1: unknown file attribute: \n
```

**Comment by RasPhilCo** [comment]
Adding this to the backlog to debug. Thanks for reporting.

**Comment by childish-sambino** [comment]
This was caused by some backticks in our command and flag descriptions. I replaced them with single-quotes and it's working properly for zsh. Recommend leaving this open and have the plugin do the string substitution.

**Comment by ArchangelX360** [comment]
According to @childish-sambino comment, this will be resolved in #12

**Comment by RasPhilCo** [comment]
Closed via #12
